packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -245,7 +245,7 @@ module.exports = ctx => ({
     ],
 
     forum_url: 'https://devforum.okta.com/',
-    copyright_text: 'Copyright © 2021 Okta.'
+    copyright_text: 'Copyright © 2022 Okta.'
```

Comment:
Would we be against doing this instead? We wouldn't have to ever think about it again 😅 
```suggestion
    copyright_text: `Copyright © ${new Date().getUTCFullYear() || '2022'} Okta.`
```

@shawyu-okta

Reply:
LGTM 🚀 !

Reply:
Great idea. Testing locally right now.

Reply:
Looks good. Will commit the suggestion. Thanks @francoislehoux-okta